_includes/partials/content/forms/dvtprogram-application-2023.html

```diff
@@ -0,0 +1,369 @@
+            <div class="form-check ms-1">
+              <input class="form-check-input applicantHardware" type="radio" name="applicantHardware" value="Yes" id="applicantHardware1">
+              <label class="form-check-label" for="applicantHardware1">
+                Yes
+              </label>
+            </div>
+            <div class="form-check ms-1">
+              <input class="form-check-input applicantHardware" type="radio" name="applicantHardware" value="No" id="applicantHardware2">
+              <label class="form-check-label" for="applicantHardware2">
+                No
+              </label>
+            </div>
+            <div class="form-check ms-1">
+              <input class="form-check-input applicantHardware" type="radio" name="applicantHardware" value="__other_option__" id="applicantHardware3">
+              <label class="form-check-label" for="applicantHardware3">
+                Other
+              </label>
+              <input type="text" class="form-control form-field d-inline" id="applicantHardwareOther">
+            </div>
+          </div>
+        </div>
+
+        <!-- Interest -->
+        <div class="w-100 mb-3">
+          <label for="applicantInterest" class="form-label">
+            <span class="text-danger">*</span>
+            Please link anything that showcases your abilities or your enthusiasm for blockchain. This can be a Github profile, a project, your blog, Twitter, your Reddit profile, etc. The more the better!
+          </label>
+          <textarea type="textarea" class="form-control form-field" id="applicantInterest" rows="5" required onchange="validate(this)"></textarea>
+        </div>
+
+        <!-- Skill -->
+        <div class="w-100 mb-3">
+          <label for="applicantSkills" class="form-label">
+            We want participants to have the aptitude or skill to complete the program. You don't need to be a programmer, but you do need to feel confident in your ability to learn and problem-solve
+            <br><br>
+            If you don't have links that showcase aptitude or ability, we prepared a smol mission for you to complete: [<a href="https://stakesaurus.notion.site/DVT-Home-Staker-Program-Aptitude-Test-9c8264a7dd0a4f088c00c5cdc09f2a56?pvs=4" target="_blank">LINK</a>]
+            Please upload the deliverables to any image-hosting website and link here. This will boost your application!
+            <br><br>
+            If you linked a Github or anything that shows existing technical skills, feel free to skip this step
+          </label>
+          <textarea type="textarea" class="form-control form-field" id="applicantSkills" rows="5" onchange="validate(this)"></textarea>
+        </div>
+
+        <!-- Address -->
+        <div class="w-100 mb-3">
+          <label for="applicantAddress" class="form-label">
+            Optional: POAP wallet address (doesn't affect your application)
```

Comment:
What's a POAP wallet address? We know what it is but I'm guessing many people don't. This needs to be explained or rephrased. It could be something like *Walllet address where you want to receive your [POAP](https://poap.xyz/)*. At this point, this question might even be entierely removed unless we want to give a POAP to everyone who applies. We'll have other opportunities to give a POAP to those who complete the program.